src/main/java/com/internet/shop/controller/InjectDataController.java

```diff
@@ -46,6 +52,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         productService.create(inspire);
 
         shoppingCartService.create(new ShoppingCart(bob.getId(), new ArrayList<>()));
+        shoppingCartService.create(new ShoppingCart(mainAdmin.getId(), new ArrayList<>()));
```

Comment:
admin should not have a shopping cart

Reply:
If one person is an admin and user, this person have to have 2 accounts?

Reply:
And why we have ```Set<Role>``` then if one user have only one role?

Reply:
@ilya449 yes, If one person is an admin and a user, this person needs to have 2 accounts
Set<Role> is for the future when we would want to add some additional roles that can be combined with admin